packages/replay/test/unit/util/createReplayEnvelope.test.ts

```diff
@@ -59,9 +59,10 @@ describe('createReplayEnvelope', () => {
             event_id: REPLAY_ID,
             platform: 'javascript',
             replay_id: REPLAY_ID,
+            replay_type: 'error',
             sdk: { integrations: ['BrowserTracing', 'Replay'], name: 'sentry.javascript.browser', version: '7.25.0' },
             segment_id: 3,
-            tags: { errorSampleRate: 0, replayType: 'error', sessionSampleRate: 1 },
+            tags: { errorSampleRate: 0, sessionSampleRate: 1 },
```

Comment:
Also side note: I think eventually, we'll want to remove these tags as well, right? IMO they don't really add value for our users and SDKs shouldn't just add additional tags to events (unless there's a justified reason to do so ofc). That being said, I know that it's convenient for us to have this data so we might want to think about a similar approach for the sample rate tags. WDYT?

Reply:
Yep I have a follow-up here: https://github.com/getsentry/sentry-javascript/pull/6659